soweego/ingester/wikidata_bot.py

```diff
@@ -162,25 +178,28 @@ def identifiers_cli(catalog, entity, identifiers, sandbox):
 
     claim (Richard Hell, Discogs artist ID, 266995)
 
-    reference (based on heuristic, artificial intelligence),
-              (retrieved, today)
+    reference (based on heuristic, artificial intelligence), (retrieved, today)
     """
-    if sandbox:
-        LOGGER.info('Running on the Wikidata sandbox item ...')
-
     add_identifiers(json.load(identifiers), catalog, entity, sandbox)
 
 
 @click.command()
 @click.argument('catalog', type=click.Choice(SUPPORTED_TARGETS))
 @click.argument('statements', type=click.File())
+@click.option(
+    '-c',
+    '--criterion',
+    type=click.Choice(('links', 'bio')),
+    help='Validation criterion used to generate STATEMENTS. '
+    'Same as the command passed to `python -m soweego sync`',
+)
 @click.option(
     '-s',
     '--sandbox',
     is_flag=True,
     help=f'Perform all edits on the Wikidata sandbox item {vocabulary.SANDBOX_2}.',
 )
-def people_cli(catalog, statements, sandbox):
+def people_cli(catalog, statements, criterion, sandbox):
```

Comment:
there seems to be significant duplication among people_cli/works_cli and add_works_statements/add_people_statements, as well as other functions here (_add_or_reference/_add_or_reference_works) is it worth it to abstract a little to make future changes easier?

Reply:
I was thinking the same: these functions were implemented in version 1.
I will reserve a new PR to refactor those parts.

Reply:
See #420